### DISCUSSION TITLE
Volume² 1.1.8.461

### DISCUSSION THREAD

**Discussion by irzyxa** [irzyxa/Volume2#250] [General]
Fixed issue related to context menu in Volume2PM 
Updated language files

<hr /><em>This discussion was created from the release <a href='https://github.com/irzyxa/Volume2/releases/tag/v1.1.8.461'>Volume² 1.1.8.461</a>.</em>

**Comment by charlestek** [comment]
Irzyxa,
Is it possible to configure Volume2 to set the default communication device?
Thanks.

**Comment by irzyxa** [comment]
There is one option:
![image](https://user-images.githubusercontent.com/5119360/220548363-a68c447b-85ed-47a5-aad4-f3f1a1b1fe69.png)
![image](https://user-images.githubusercontent.com/5119360/220548553-4699016b-0784-426e-83d2-2d9e8f20280b.png)

**Comment by charlestek** [comment]
Ah, I see, thank you very much.

**Comment by wsdyma** [comment]
可不可以将音量峰值仪表置入于系统托盘？
以下是机器翻译
Can I put the volume peak meter in the system tray?

**Reply by irzyxa** [reply]
So you would like to use it separately, right? Because now it is part of Volume2

**Comment by derfux11** [comment]
Bonjour je me permets d'amener ma petite contribution au truc bidule
Attention les amis il y a un bon gros virus à l'intérieur DE Volume2_1_1_7_449.zip
Sur GitHub c'est inadmissible j'aurais jamais cru 
Regardez par vous-même sur VirusTotal
https://urlz.fr/lfwu
Le deuxième fichier Setup_Volume2_1_1_7_449.zip ne contient pas de virus selon Virus Total d'ailleurs c'est pas un virus c'est un Trojan.Generic.xqar
Jiangmin
Trojan.Generic.xqar
MaxSecure
Cheval de Troie.Malware.300983.susgen
J'espère que c'est un faux positif et pas vraiment un virus c'est bizarre qu'il n'y a que moi qui m'en soit aperçu je passe tout ce que je télécharge a Virus Total

**Reply by irzyxa** [reply]
It is known issue because Volume2 set hooks for keyboard and mouse to control volume so some antiviruses think it has virus

**Reply by irzyxa** [reply]
Also the same files were uploaded to Microsoft Store and Steam and there was not any issues.

**Comment by donatopace** [comment]
Hello,
after many years!
There is a solution to change volume of  VOICEMEETER POTATO?
I mean with mouse wheel as actually volume 2 do?
My computer has default device VOICEMEETER VAIO3
and to change volume the only solution i have found is 
Audio device: Altoparlanti Realtek
(see picture attached)

Motivation?
When i have several audio devices i cannot change global volume but with VOICEMEETER VAIO 3 yes, but it is not compatible as default device with Volume 2.
Thanks for your work.

![Volume2 Setting](https://user-images.githubusercontent.com/43314459/233410110-4f625491-ea7e-4208-9473-5e32937efa91.png)

**Comment by donatopace** [comment]
Possible solution that I found could be to hook directly on the "FADER GAIN" of VAIO3?
If there is a lot of work i can send you money for this extra, thanks.
Thanks

**Comment by SkyTrax79** [comment]
Hi, maybe I just can't find where is the option but is there a way to control the alternative device without using the keyboard ?  Something like rotating the mouse wheel when holding the left button, or rotating the wheel on a specific screen edge for the alternative device.  My goal was to control 2 audio devices when i'm far away from it, if I had the keyboard just beside me it would be more simple to just use the media keys or whatever comes with the keyboard.

But thanks anyway for the program, it's appreciated and it already helps a lot.

**Reply by irzyxa** [reply]
I see only one way:
![image](https://github.com/irzyxa/Volume2/assets/5119360/29435432-2193-41dd-acb7-00e900e3a063)

**Reply by SkyTrax79** [reply]
Thanks for the suggestion, what i'm trying to achieve is when I rotate the mouse wheel on the taskbar it changes the volume of the primary device and when I rotate the mouse on the top egde of the screen it changes the volume of the alternative device only.

**Comment by Adeptuzzz** [comment]
Drweb detects Adware.Conduit.591
